e2e-tests/IIoTPlatform-E2E-Tests/TestHelper.cs

```diff
@@ -471,16 +480,119 @@ private static PrivateKeyFile GetPrivateSshKey(IIoTPlatformTestContext context)
                     }
 
                     Assert.NotEmpty(response.Content);
-                    json = JsonConvert.DeserializeObject(response.Content);
+                    json = JsonConvert.DeserializeObject<ExpandoObject>(response.Content, new ExpandoObjectConverter());
```

Comment:
ExpandedObject?

Reply:
Avoid an error in Newtonsoft.Json deserialization, ExpandoObject are dynamic classes  were the calls to the object are verified at runtime. [MSDON ExpandoObject](https://docs.microsoft.com/en-us/dotnet/api/system.dynamic.expandoobject?view=netcore-3.1)